smoke-tests/src/test/groovy/io/opentelemetry/smoketest/CrashEarlyJdk8Test.groovy

```diff
@@ -17,10 +16,14 @@ import org.testcontainers.utility.MountableFile
 import spock.lang.IgnoreIf
 import spock.lang.Specification
 
+import java.time.Duration
```

Comment:
Formatting looks off

Reply:
I think this is the correct formatting (#1708)